mmfunctions

```diff
@@ -174,6 +175,15 @@ _premis_event_list(){
     fi
 }
 
+_generate_fingerprint(){
+    if [ ${PERCEPTUAL_HASH} = "1" ] ; then
+        SIGNATURE="$(basename ${INPUT%.*})_signature.xml"
```

Comment:
%.* needs to be removed to match with current conventions of other sidecar files